experimental/gittuf/targets.go

```diff
@@ -259,6 +291,14 @@ func (r *Repository) RemoveDelegation(ctx context.Context, signer sslibdsse.Sign
 		return err
 	}
 
+	if signCommit {
+		slog.Debug("Checking if git signing is configured...")
```

Comment:
move this above signer.KeyID? in fulcio, this would trigger an oidc flow to get the keyID.